src/bci_build/package/apache_tomcat.py

```diff
@@ -0,0 +1,81 @@
+"""Provide an Apache Tomcat container."""
+
+from bci_build.package import CAN_BE_LATEST_OS_VERSION
+from bci_build.package import DOCKERFILE_RUN
+from bci_build.package import _SUPPORTED_UNTIL_SLE
+from bci_build.package import OsVersion
+from bci_build.package import ParseVersion
+from bci_build.package import Replacement
+
+from .appcollection import ApplicationCollectionContainer
+
+_TOMCAT_VERSIONS: list[int] = [9, 10]
+assert _TOMCAT_VERSIONS == sorted(_TOMCAT_VERSIONS)
+
+TOMCAT_CONTAINERS = [
```

Comment:
Can we **please** stop moving containers around between files? It makes blame & log needlessly annoying.

Reply:
it no longer makes sense in appcontainers because those are the "now legacy" application containers. and imho each container should be in its own file. as we had to move it anyway, splitting it into its separate file ensures that annotation/blaming will continue going forward. 

btw, there is `--follow-renames`, fjyi

Reply:
I agree that each container should be its own file, it is easier to manage in the long term. As I see, we are entering a new chapter of the project which justifies such changes.

Reply:
Brilliant, can you please then move the code in a separate commit than the changes? It at least makes reviews simpler.

Reply:
splitted into two commits